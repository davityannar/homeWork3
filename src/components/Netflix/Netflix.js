import React from 'react';
import './Netflix.scss';

const Netflix = () => {
    return(
        <p className='netflix'>
            netflixroulette
        </p>
    )
}

export default Netflix;