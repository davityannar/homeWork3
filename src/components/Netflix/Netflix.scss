.netflix {
    margin: 25px 0;
    padding-top: 50px;
    color: #f55263;
}