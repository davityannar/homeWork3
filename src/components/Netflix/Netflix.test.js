import React from 'react';
import Netflix from './Netflix';

describe('Tests Netflix component', () => {
  it('Matching snapshot', () => {
    const component = shallow(<Netflix />);

    expect(component).toMatchSnapshot();
  })
})