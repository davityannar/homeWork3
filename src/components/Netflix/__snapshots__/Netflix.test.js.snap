// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Netflix component Matching snapshot 1`] = `
<p
  className="netflix"
>
  netflixroulette
</p>
`;
